### DISCUSSION TITLE
Google Mobile Ads SDK 7.66.0 for iOS support

### DISCUSSION THREAD

**Discussion by lucasaugustin** [invertase/react-native-firebase#4387] [Q&A] [answered]
Hey guys.
Does the latest version of react-native-firebase/adMob (7.6.8) uses the version 7.66.0 of GMA SDK for iOS?
Or has it been updated some how to handle iOS 14 changes?

**Comment by mikehardy** [comment, marked as answer]
https://github.com/invertase/react-native-firebase/issues/4107

**Reply by lucasaugustin** [reply]
thaanks
